docs/modules/ROOT/pages/advanced-scripting.adoc

```diff
@@ -537,73 +537,6 @@ Fun 1
 
 == Output
 
-=== Type of CLC Output
-
-CLC supports two kinds of output: structured and "unnecessary".
-Let's talk about them over an example.
-
-`clc demo generate-data` command creates data in a Map or to the terminal.
-Let's run it and analyze its output:
-
-[source,bash]
-----
-clc demo generate-data wikipedia-event-stream --preview --yes --max-values 1
-----
-
-Running that command generates output similar to the following:
-----
-Following mapping will be created when run without preview:
-
-CREATE OR REPLACE MAPPING "<map-name>"
-(
-    -- ...SNIP...
-);
-
--------------------------------------------------------------------------------------------------------------------------------------------
- ID                               | Timestamp                        | User                             | Comment
--------------------------------------------------------------------------------------------------------------------------------------------
- aa1d8f0b-2218-4ac7-aaec-727eb109 | 2023-11-03T10:07:39Z             | Dcirovicbot                      | /* wbeditentity-update-languages
- 6e25                             |                                  |                                  | -short:0||bs */
--------------------------------------------------------------------------------------------------------------------------------------------
-----
-
-In the output, the table is the structured output and the rest is "unnecessary" output.
-Of course the information in the "unnecessary" output is not really unnecessary, but it is not as important as the structured output and you can get rid of that using the `-q` flag:
-
-[source,bash]
-----
-clc demo generate-data wikipedia-event-stream --preview --yes --max-values 1 -q
-----
-
-Now the output contains only the structured output:
-----
--------------------------------------------------------------------------------------------------------------------------------------------
- ID                               | Timestamp                        | User                             | Comment
--------------------------------------------------------------------------------------------------------------------------------------------
- aa1d8f0b-2218-4ac7-aaec-727eb109 | 2023-11-03T10:07:39Z             | Dcirovicbot                      | /* wbeditentity-update-languages
- 6e25                             |                                  |                                  | -short:0||bs */
--------------------------------------------------------------------------------------------------------------------------------------------
-----
-
-You can easily change the format of the structured output by using the `-f` flag, such as JSON:
```

Comment:
You can change the format of the structured output using the `-f` flag, as shown in the following JSON example: